components/atoms/BenefitCardHeaderActive.js

```diff
@@ -36,11 +36,11 @@ export default function BenefitCardHeaderActive(props) {
         <HorizontalRule width="w-1/3" visibility="sm:hidden" />
         <div
           id={`${props.benefit.benefitType}-active-paymentStartDate`}
-          className="font-display grid-cols-3 col-span-3 py-4 sm:grid sm:pl-10"
+          className="grid col-span-3 gap-y-4 gap-x-1 sm:grid-cols-3 sm:pl-8 lg:pl-10 font-display"
```

Comment:
It seems we're loosing some padding at the top and bottom of these details on mobile...

Reply:
I discussed it with Claudia and that is what she wanted it seems

Reply:
okay :)